contrib/dpu_daemon/scripts/build_x86.sh

```diff
@@ -1,122 +1,121 @@
-#!/bin/bash
-set -e
-
+#!/bin/bash -eE
 arch=x86
```

Comment:
Consider using `set -o pipefail` and `trap ERR`

Reply:
There are no pipes